.github/workflows/docx-ci.yml

```
@@ -0,0 +1,48 @@
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.11'
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 19: could not find expected ':'

(syntax-check)